### PR TITLE
[FEAT] Add a new example to customize the style of a new overlay

### PR DESCRIPTION
**Pull request by aibcmars**
relates to:
https://github.com/process-analytics/bpmn-visualization-js/issues/1172

https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/overlays-styling_interface/examples/index.html

### CODE REVIEW

**Comment by aibcmars** [comment]
image
image

**Comment by csouchet** [comment]
To separate the feature to position the overlay & the feature to style the overlay, I would remove the Position buttons and I choose arbitrary (like we done in the element-identification page) a position for the created overlay.

**Comment by csouchet** [comment]
### After changes
image
image

File: examples/overlays/add-remove/index.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
 <html lang="en">
     <head>
         <meta charset="UTF-8">
-        <title>BPMN Visualization - Diagram Navigation</title>
+        <title>BPMN Visualization - Add & remove default overlays</title>
```
Comment (tbouffard) [inline examples/overlays/add-remove/index.html:18]: 👍🏿 good catch

File: examples/overlays/add-remove/index.html [hunk 2]
```diff
@@ -109,7 +109,7 @@ <h2>Add & remove default overlays</h2>
                                     </g>
                                 </svg>
                             </button>
-                            <button id="btn-end" type="button" class="btn bg-secondary"title="Add overlay on the end of the edge">
+                            <button id="btn-end" type="button" class="btn bg-secondary" title="Add overlay on the end of the edge">
```
Comment (tbouffard) [inline examples/overlays/add-remove/index.html:112]: 👍🏿 good catch

File: examples/overlays/add-stylized/index.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+// Load BPMN diagram
+bpmnVisualization.load(getGettingStartedBpmnDiagram(), { fit: { type: 'Center' } });
+
+document.getElementById('btn-set-overlay').onclick = () => {
+    removeAllOverlays();
+
+    // Add overlays
+    // Shape
+    bpmnElementsRegistry.addOverlays('Activity_1potg3p', { position: 'bottom-right', label: 'NaN', style });
```
Comment (tbouffard) [inline examples/overlays/add-stylized/index.js:47]: ❓ `NaN` may be confusing as it often show an overfow. Can we use something else?

**Review by tbouffard — APPROVED** [review]
LGTM, a minor stuff could be done to avoid displaying `NaN`

File: examples/overlays/add-stylized/index.js (hunk 3 above)
Comment (csouchet) [reply examples/overlays/add-stylized/index.js:47]: Yes, but I don't have any idea. 
I just wanted to show that we can use **text** and not just **number** as value.

If you have a idea, you can change & merge 😉

Comment (tbouffard) [reply examples/overlays/add-stylized/index.js:47]: we could put`OK`? or something with an emoji as well?

**Comment by tbouffard** [comment]
### Proposals for NaN replacements

#### text only

image

#### with emoji

on firefox

image

on chrome

image

**Pull request merged**